src/libnet_link_dlpi.c

```diff
@@ -11,20 +11,28 @@
  *	The Regents of the University of California.  All rights reserved.
```

Comment:
This is partially copyright the Un of Calif, do you have their agreement to relicense this file? I suspect not personally :-), but if you can find the original source you may be able to find an updated version of it with a new license. In the absence of some evidence, you can't legally change the license without agreement on the explicit and implicit (because of donation)  owners of the copyright.

Reply:
actually, I take it back, I see they gave blanket permission :+1:

Reply:
Yeah, that blanket statement is what just about every other BSD-4 project has cited when changing to BSD-3.  I hope you are comfortable resolving this conversation/review?